### DISCUSSION TITLE
MySQL settings

### DISCUSSION THREAD

**Discussion by andy1463** [1Remote/1Remote#443] [Q&A]
Hello,
is there some information regarding MySQL database setup (table settings, kollation etc.)?

Thanks a lot
Andreas

**Comment by VShawn** [comment]
you can check this: http://1remote.org/usage/team/team-sharing/#why

**Reply by andy1463** [reply]
Thank you. But do I only need database and users or do I have to create some tables? If yes which tables?

Thanks

**Reply by VShawn** [reply]
> Thank you. But do I only need database and users or do I have to create some tables? If yes which tables?
> 
> 
> 
> Thanks

u need create one database, then fill the database and db user & password into App(make sure it's writable), tables will be created by App automatically

**Reply by andy1463** [reply]
Thanks a lot
